packages/sandpack-core/src/transpiled-module/transpiled-module.ts

```diff
@@ -1152,13 +1117,8 @@ export class TranspiledModule {
       source: null,
     };
 
-    const isNpmDependency = this.module.path.startsWith('/node_modules/');
-    const canOptimizeSize = sourceEqualsCompiled && optimizeForSize;
-    // Don't cache source if it didn't change, also don't cache changed source from npm
-    // dependencies as we can compile those really quickly.
-    const shouldCacheTranspiledSource = !canOptimizeSize && !isNpmDependency;
-
-    if (shouldCacheTranspiledSource) {
+    if (!sourceEqualsCompiled) {
```

Comment:
Do you want to save transpiled code for npm dependencies as well? I think this could grow the cache quite a bit.

Reply:
We kind of have to, otherwise it assumes it does not need transpilation

Reply:
But will revert it, the issues are a bit deeper than this anyway, gonna hopefully catch them once I start rewriting the transpilation logic.